interceptors/payable_resource_authentication.go

```diff
@@ -66,29 +65,9 @@ func (payableAuthInterceptor *PayableAuthenticationInterceptor) PayableAuthentic
 			}
 		}
 
-		// Get the payable resource from the ID in request
-		payableResource, responseType, err := payableAuthInterceptor.Service.GetPayableResource(r, companyNumber, payableID)
```

Comment:
Attempt to reduce Code complexity by grouping simile logic checks into one Go function